lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -350,7 +350,12 @@ def get_parent_of(record, path)
   def send_bulk(data)
     retries = 0
     begin
-      client.bulk body: data
+      response = client.bulk body: data
+      if response['errors']
+        message = "Could not push log to Elasticsearch.\n"
```

Comment:
Multiline logs aren't fun to parse, let's simplify this to `"Could not push log to Elasticsearch: #{response}"` so as much information is available as can be for debug purposes